dockerize/Dockerfile

```diff
@@ -102,13 +86,25 @@ COPY dockerize/supervisord.conf /etc/
 # Copy the rest of B2Share files
 #
 
-WORKDIR /eudat/b2share
-ADD webui webui
-WORKDIR /eudat/b2share/webui
-RUN node_modules/webpack/bin/webpack.js -p
-
 WORKDIR /eudat
 ADD . b2share
```

Comment:
Is this intentionally here after line 75 & 76?
If I understood the Dockerfile content correctly, line 76 installs B2SHARE as editable from a folder which does not contain b2share source code at the time of install.

Reply:
Yes it is intentional. It's the same as it was in the orginal b2share dockerfile https://github.com/EUDAT-B2SHARE/b2share/blob/master/dockerize/Dockerfile#L86